keyboards/ckeys/thedora/readme.md

```diff
@@ -32,7 +32,15 @@ All cKeys PCBs are open source. However, this one has not been posted yet becaus
 
 If you want to reflash the pre-installed firmware, use the `.bin` file for Proton C. Flash with the QMK Toolbox or via the command line.
 
-Building for Proton C: `make ckeys/thedora:default`
+**Building for Proton C:**
+`make ckeys/thedora:default`
+
+**Enter DFU-Bootloader Mode:**
+`make ckeys/thedora:default:dfu-util`
+
+**Flash Firmware:**
```

Comment:
And kinda as above, this process is what actually triggers the device to go into the bootloader.